lib/hydra/derivatives/image.rb

```diff
@@ -53,10 +54,19 @@ def write_image(output_file, xfrm)
         output_file.content = stream
       end
 
-      # Override this method if you want a different transformer, or need to load the
-      # raw image from a different source (e.g.  external file)
+      # Override this method if you want a different transformer, or # need to load the raw image from a different source (e.g.  
```

Comment:
Should these changes be in the `Image` class, or in an overridden `#load_image_transformer` method in the `RawImage` class?